### ISSUE TITLE
Difference migration feature request

### ISSUE THREAD

**Issue by mehmetdemiray** [tomjschuster/sequelize-ui#94]
App is migrating everything as combined currently. My feature request is separated migrations. For example I created new scheme and it created migrations it's ok. After that if i add new field in model it needs to create new migration.

Maybe one more button needs instead of "save" and "save with new migration". If you click save button migrations appear combined. If you click migrate button it create new migration according to the difference with the last changes. Becouse as you know after start the project we can not changes migrations only we can add new ones for changes.

**Comment by tomjschuster** [comment]
I agree that currently the generated migrations are only useful for new projects/tables;  this request makes sense.

I like the "Save Migration" idea and would love to dig into this when I have some time.
